### PR TITLE
KP decompress vs uncompress

### PR DESCRIPTION
**Pull request by intelkevinputnam**
Resolved misuse of the word "uncompress" by replacing it with "decompress" where it made sense. This also involved:

1. Changing some file names
2. Fixing some includes
3. Updating TOC trees

Also opportunistically fixed:

1. |CL-ATTR| where needed
2. typo in mixer.rst which was causing an anchor not to be found

### CODE REVIEW

File: source/clear-linux/guides/maintenance/download-verify-decompress-linux.rst [hunk 1]
```diff
@@ -1,15 +1,15 @@
-.. _download-verify-uncompress-linux:
+.. _download-verify-decompress-linux:
 
-Download, verify, and uncompress a Clear Linux image on Linux
+Download, verify, and decompress a Clear Linux image on Linux
```
Comment (krisdale) [inline source/clear-linux/guides/maintenance/download-verify-decompress-linux.rst]: Can update the "Clear Linux" in page title to use |CL-ATTR| sub. (First use on page)

File: source/clear-linux/guides/maintenance/download-verify-decompress-linux.rst [hunk 2]
```diff
@@ -45,24 +45,28 @@ the screen followed by `OK`.
 For a more in-depth discussion of image verification including checking the
 certificate see :ref:`image-content-validation`.
 
-Uncompress the Clear Linux image
+.. incl-decompress-image:
+
+Decompress the Clear Linux image
```
Comment (krisdale) [inline source/clear-linux/guides/maintenance/download-verify-decompress-linux.rst]: After use of |CL-ATTR|, subsequent use of "Clear Linux" can be replaced with |CL| sub.

File: source/clear-linux/guides/maintenance/download-verify-decompress-mac.rst [hunk 3]
```diff
@@ -1,15 +1,15 @@
-.. _download-verify-uncompress-mac:
+.. _download-verify-decompress-mac:
 
-Download, verify, and uncompress a Clear Linux image on macOS\*
+Download, verify, and decompress a Clear Linux image on macOS\*
```
Comment (krisdale) [inline source/clear-linux/guides/maintenance/download-verify-decompress-mac.rst]: Can opportunistically update "Clear Linux" text in titles and text with subs.

File: source/clear-linux/guides/maintenance/download-verify-decompress-windows.rst [hunk 4]
```diff
@@ -1,15 +1,15 @@
-.. _download-verify-uncompress-windows:
+.. _download-verify-decompress-windows:
 
-Download, verify, and uncompress a Clear Linux image on Windows\*
+Download, verify, and decompress a Clear Linux image on Windows\*
```
Comment (krisdale) [inline source/clear-linux/guides/maintenance/download-verify-decompress-windows.rst]: Can opportunistically update "Clear Linux" text in titles and text with subs.

File: source/clear-linux/tutorials/web-server-install/web-server-install.rst [hunk 5]
```diff
@@ -3,7 +3,7 @@
 Create a Clear Linux\* based web server
 #######################################
```
Comment (krisdale) [inline source/clear-linux/tutorials/web-server-install/web-server-install.rst]: Can opportunistically update "Clear Linux" text in titles and text with subs.

File: source/clear-linux/guides/maintenance/mixer.rst [hunk 6]
```diff
@@ -4,7 +4,7 @@ Use the mixer tool
 ##############
```
Comment (krisdale) [inline source/clear-linux/guides/maintenance/mixer.rst]: Correct title underline (giving a warning)

File: source/clear-linux/guides/maintenance/download-verify-decompress-windows.rst (hunk 4 above)
Comment (mvincerx) [inline source/clear-linux/guides/maintenance/download-verify-decompress-windows.rst]: Replace `Clear Linux` with |CL-ATTR| here please.

File: source/clear-linux/guides/maintenance/download-verify-decompress-mac.rst (hunk 3 above)
Comment (mvincerx) [inline source/clear-linux/guides/maintenance/download-verify-decompress-mac.rst]: Replace Clear Linux with |CL-ATTR| here please.

File: source/clear-linux/guides/maintenance/mixer.rst (hunk 6 above)
Comment (mvincerx) [inline source/clear-linux/guides/maintenance/mixer.rst]: Since your PR follows those before on same issue, please add `#` to include full length here.

**Comment by intelkevinputnam** [comment]
I believe I've addressed all of the comments in addition to fixing the internal TOC in mixer.rst.

**Comment by mvincerx** [comment]
@krisdale. LMK when you approve please.

**Comment by mvincerx** [comment]
@krisdale Merging in order to hit upcoming build.

**Comment by mvincerx** [comment]
Closes Issue #241 
https://github.com/clearlinux/clear-linux-documentation/issues/241

**Pull request merged**